### ISSUE TITLE
[AWS::ECS::Cluster] - [Enhancement] - Implement `GetAtt` for the `Resource Name`

### ISSUE THREAD

**Issue by r-heimann** [aws-cloudformation/aws-cloudformation-coverage-roadmap#1611]
### Name of the resource

AWS::ECS::Cluster

### Description

Currently `AWS::ECS::Cluster` only supports `Ref` to get the `Resource Name`. Implementing this would be great.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-cluster.html

Related to https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/68

**Comment by r-heimann** [comment]
Still no `!GetAtt` support

**Comment by r-heimann** [comment]
Still no `!GetAtt` support

**Comment by s3cube** [comment]
We currently have support for `!GetAtt` for the Arn: https://docs.aws.amazon.com/AWSCloudFormation/latest/TemplateReference/aws-resource-ecs-cluster.html#aws-resource-ecs-cluster-return-values

**Comment by r-heimann** [comment]
Yes that is right, but the `!Ref` value cannot be retrieved using something like `!GetAtt ECSCluster.Name` which is this feature request.
Every `!Ref` value should also be available as a `!GetAtt`, because every CloudFormation Resource has different values for `!Ref` and it is not human readable.

**Comment by r-heimann** [comment]
Still no `!GetAtt` support

**Comment by r-heimann** [comment]
Still no `!GetAtt` support
